libraries/core_libs/consensus/src/pbft/block_proposer.cpp

```diff
@@ -179,7 +179,7 @@ void BlockProposer::proposeBlock(DagFrontier&& frontier, level_t level, SharedTr
   // simply add the block to the DAG
   DagBlock blk(frontier.pivot, std::move(level), std::move(frontier.tips), std::move(trx_hashes), std::move(vdf),
                node_sk_);
-  dag_mgr_->addDagBlock(blk, std::move(trxs), true);
+  dag_blk_mgr_->pushVerifiedBlockToFront(blk);
```

Comment:
I think trxs is not needed anymore, maybe we could change `SharedTransactions shared_trxs = proposer->getShardedTrxs();` to return just hashes?

Reply:
I have made changes so this does not longer apply